xml/System/Environment.xml

```diff
@@ -2493,21 +2493,21 @@ The following example creates environment variables for the <xref:System.Environ
         <ReturnType>System.Version</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a version consisting of the major, minor, build, and revision numbers of the common language runtime.</summary>
-        <value>The version of the common language runtime.</value>
+        <summary>Gets a version consisting of the major, minor, build, and revision numbers of the Common Language Runtime.</summary>
+        <value>The version of the Common Language Runtime.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- For the .NET Framework Versions 4, 4.5, 4.5.1, and 4.5.2, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns a <xref:System.Version> object whose string representation has the form `4.0.30319.xxxxx`. For the .NET Framework 4.6 and later versions, it has the form `4.0.30319.42000`.
+ For the .NET Framework Versions 4, 4.5, 4.5.1, and 4.5.2, the <xref:System.Environment.Version%2A?displayProperty=nameWithType> property returns a <xref:System.Version> object whose string representation has the form `4.0.30319.xxxxx`. For the .NET Framework 4.6 and later versions, and .NET Core versions before 3.0, it has the form `4.0.30319.42000`. For .NET Core 3.0 and later versions, this property returns the actual .NET Core version number.
 
 > [!WARNING]
-> For the .NET Framework 4.5 and later, we do not recommend using the <xref:System.Environment.Version%2A> property to detect the version of the runtime; instead, you can determine the version of the common language runtime by querying the registry. For more information, see [How to: Determine Which .NET Framework Versions Are Installed](/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed).
+> For the .NET Framework 4.5 and later, we do not recommend using the <xref:System.Environment.Version%2A> property to detect the version of the runtime; instead, you can determine the version of the Common Language Runtime by querying the registry. For more information, see [How to: Determine Which .NET Framework Versions Are Installed](/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed).
 
- For more information about the version of the common language runtime that is installed with each version of the .NET Framework, see [Versions and Dependencies](/dotnet/framework/migration-guide/versions-and-dependencies).
+ For more information about the version of the Common Language Runtime that is installed with each version of the .NET Framework, see [Versions and Dependencies](/dotnet/framework/migration-guide/versions-and-dependencies).
 
 ## Examples
- The following example displays the version of the common language runtime. (The version is omitted from the example output for security reasons.)
+ The following example displays the version of the Common Language Runtime. (The version is omitted from the example output for security reasons.)
```

Comment:
```suggestion
 The following example displays the version of the Common Language Runtime.
```
I do not get what are the security reasons that this is talking about.

Reply:
I also don't. Probably the author intended to warn that publishing runtime version from your machine could have security concerns (i.e. if for some reason you have old runtime without patches). [MachineName property](https://docs.microsoft.com/ru-ru/dotnet/api/system.environment.machinename?view=netcore-2.0#examples) has such note as well. But here it is out of place, because example does not even have any output shown, so will remove it.